internal/ui/main_page.go

```diff
@@ -104,44 +76,54 @@ func (a *App) MainPage(ctx context.Context) error {
 		SetHorizontal(true)
 
 	form.SetBorder(true).
-		SetTitle("Find DotA2 Match ID").
+		SetTitle("find dota 2 match id").
 		SetTitleAlign(tview.AlignCenter)
 
-	layoutListDownload.
-		AddItem(txtDownloads, 0, 1, false).
-		AddItem(listDownload, 0, 2, false)
-	layoutListDownload.SetBorder(true)
+	downloads := tview.NewFlex().
+		SetDirection(tview.FlexRow)
+	downloadsTitle := tview.NewTextView().
+		SetText("downloads").
+		SetTextAlign(tview.AlignLeft)
+	downloadsList := tview.NewList()
 
-	rootLayout := tview.NewFlex().SetDirection(tview.FlexRow).
-		AddItem(textView, 0, 1, false).
-		AddItem(form, 0, 2, false).
-		AddItem(layoutListDownload, 0, 2, false)
+	replays := a.getDownloadedReplays()
+	for _, replay := range replays {
+		// TODO:
+		// add binding to `selected func` redirect to new page
+		// which will display a replay
+		downloadsList.AddItem(replay, "", '-', nil)
+	}
 
-	err = a.Application.SetRoot(rootLayout, true).SetFocus(form).Run()
+	downloads.
+		AddItem(downloadsTitle, 0, 1, false).
+		AddItem(downloadsList, 0, 2, false).
+		SetBorder(true)
 
-	if err != nil {
-		return err
-	}
+	root := tview.NewFlex().
+		SetDirection(tview.FlexRow).
+		AddItem(title, 0, 1, false).
+		AddItem(form, 0, 2, false).
+		AddItem(downloads, 0, 2, false)
```

Comment:
title, form, and downloads are instantiated in order for better reading experience